test/parallel/test-fs-read.js

```diff
@@ -21,10 +21,11 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const path = require('path');
 const fs = require('fs');
-const filepath = path.join(common.fixturesDir, 'x.txt');
+const filepath = path.join(fixtures.fixturesDir, 'x.txt');
```

Comment:
There is the dedicated `fixtures.path` function to make this even shorter. It would be nice to use that instead.